mutiny-core/Cargo.toml

```diff
@@ -15,7 +15,7 @@ repository = "https://github.com/mutinywallet/mutiny-node"
 [dependencies]
 ln-websocket-proxy = "0.3.0"
 lnurl-rs = { version = "0.2.2", default-features = false, features = ["async", "async-https"] }
-
+nostr-sdk = { version = "0.21.0", default-features = false }
```

Comment:
This uses `tokio`, are you sure it's browser safe? Might be wasm safe but idk about `tokio::sync` https://github.com/rust-nostr/nostr/blob/master/crates/nostr-sdk/Cargo.toml#L41

Reply:
I tested and was able to broadcast a transaction, will look into where tokio is needed

Reply:
looks like it is used for channels

Reply:
I didn't think tokio channels were wasm safe. I've just been going with futures.

Reply:
Nothing I'm seeing online convinces me that it's truly wasm safe. From documentation, I see that even `tokio::sync::mpsc` will block if there's multiple sends that it needs to do. https://github.com/tokio-rs/tokio/blob/c89406965ffb4a64936d781c556a2c48855dfbdb/tokio/src/sync/mpsc/bounded.rs#L448-L451

Reply:
Okay will convert to draft and we can make PRs into rust-nostr

Reply:
I could also be wrong, it's just so hard to tell

Reply:
We could just implement our own client. All we need is a websocket, won't be very hard.